### PR TITLE
Support prerelease semver in make file

### PR DESCRIPTION
**Pull request by alexhung**
Revert the ‘e’ backup extension when execute with GNU sed. Add info on GNU sed installation to README

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -190,6 +190,28 @@ Simply run `make install` - this will compile the provider and install it to `~/
 Requirements:
 - [Terraform](https://www.terraform.io/downloads.html) 0.13
 - [Go](https://golang.org/doc/install) 1.15+ (to build the provider plugin)
+- [gnu-sed](https://formulae.brew.sh/formula/gnu-sed)
+
+### gnu-sed
+
+This provider uses [GNU sed](https://www.gnu.org/software/sed/) as part of the build toolchain, in both Linux and macOS. This provides consistency across OSes. If you are building this on macOS, you will need to install [gnu-sed using brew](https://formulae.brew.sh/formula/gnu-sed).
```
Comment (chb0github) [inline README.md:197]: Honestly, why not just build with a docker image and call it done? We can dictate the terms of a build

**Comment by chb0github** [comment]
This will work - but honestly, why not just use a docker image?

File: README.md (hunk 1 above)
Comment (alexhung) [reply README.md:197]: If the user wants to build and run in macOS (like we do currently), this is still an issue.

Comment (danielmkn) [reply README.md:197]: I guess I need to do the same for Xray? :-/

**Review by danielmkn — DISMISSED** [review]

File: README.md (hunk 1 above)
Comment (alexhung) [reply README.md]: @danielmkn You should for the time being. We will pick a Docker image as the 'official' image to use for building the provider. Let's get this and the Xray provider released first before we do that.

**Review by danielmkn — APPROVED** [review]

**Pull request merged**